README.md

```diff
@@ -2,26 +2,35 @@
   <a href="https://clerk.dev?utm_source=github&utm_medium=clerk_docs" target="_blank" rel="noopener noreferrer">
     <picture>
       <source media="(prefers-color-scheme: dark)" srcset="https://images.clerk.dev/static/logo-dark-mode-400x400.png">
-      <img src="https://images.clerk.dev/static/logo-light-mode-400x400.png" height="64">
+      <img src="https://images.clerk.dev/static/logo-light-mode-400x400.png" height="120">
     </picture>
   </a>
   <br />
 </p>
 
-# Clerk Documentation
+<h1 align="center">
+  Clerk Documentation
+</h1>
 
-This repository contains the content for [Clerk's documentation](https://clerk.com/docs).
+Clerk is more than authentication, and we want to convey every feature of Clerk's as best as we can to make authentication and user management simpler for everyone. Explaining these concepts can sometimes be tricky, but with the help of a community, we can collaborate, share ideas, find the right words, and achieve greater clarity.
 
-### I've found an issue with your documentation
+We encourage you to contribute to Clerk's documentation! You can get started by following [this guide](./CONTRIBUTING.md).
```

Comment:
I think I understand what the intent is here, but I would expect this content to live under a "Contributing" or "Community" section, not necessarily the top-level blurb.

Reply:
Hmm, I was kind of taking after other docs like Astro, Solid, etc but we can simplify.

Reply:
Ah, I see. I like how Astro does it! With the separate sections:

* https://github.com/withastro/docs#we-are-astro
* https://github.com/withastro/docs#you-are-awesome